scrapy/core/scheduler.py

```diff
@@ -13,9 +12,11 @@
 
 class Scheduler(object):
 
-    def __init__(self, dupefilter, jobdir=None, dqclass=None, mqclass=None, logunser=False, stats=None):
+    def __init__(self, dupefilter, jobdir=None, logunser=False, stats=None,
+                 pqclass=None, dqclass=None, mqclass=None):
```

Comment:
Yeah, I guess if someone subclassed Scheduler and overrode `from_crawler` for example, that might bite.
Well, if you say it's unsafe for the keyword args, I'll add the `pqclass` after `stats`.

Since you say "technically backwards-incompatible", should I change it or not?
Probably it should be backwards-compatible if it goes into 1.1, but maybe not necessarily if it only hits 1.2 ?

Hmm, let me know.